### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by johnnyreilly**
Proposed in response to https://github.com/jquery/globalize/issues/494

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -214,6 +214,8 @@ requirements. See table below.
 | Plural module | cldr/supplemental/plurals.json (for cardinals)<br>cldr/supplemental/ordinals.json (for ordinals) |
 | Relative time module | cldr/main/`locale`/dateFields.json<br>+CLDR JSON files from number and plural modules |
 
+As alternative to deducing this yourself you could try using a tool called [Globalize · So What'cha Want](http://johnnyreilly.github.io/globalize-so-what-cha-want/).  This allows you to select the modules you're interested in using and tell you the Globalize files *and* CLDR JSON that you need.
```
Comment (sompylasar) [inline README.md]: This *tool* allows you ... and tell*s* you ...

Comment (rxaviers) [inline README.md]: Also note a minor details: double space `  `.

Comment (kswedberg) [inline README.md]: :+1: to @sompylasar and @rxaviers comments. 

Also:
* Change "As alternative" to "As an alternative"
* comma after "yourself"
* Change "you could try using" to  "you could use"

So the first sentence would look like this:

> As an alternative to deducing this yourself, you could use a tool called \[Globalize · So What'cha Want](http://johnnyreilly.github.io/globalize-so-what-cha-want/).

Comment (rxaviers) [inline README.md]: Thanks @kswedberg.

I'm also wondering if we can simplify the sentence... Something like this:

> As an alternative to deducing the CLDR requirements yourself, use this [online tool][]. The tool allows you ... and tells you ...

Comment (rxaviers) [inline README.md]: EDIT: simplify the link.

Comment (kswedberg) [inline README.md]: LGTM

**Comment by johnnyreilly** [comment]
Thanks for the feedback folks - the changes have been made.

**Comment by rxaviers** [comment]
:+1: thanks for your contrib.

**Comment by johnnyreilly** [comment]
My pleasure!

**Pull request closed without merging**